submissions/NwagboJohn/OOP_task/ecommerceClasses

```diff
@@ -0,0 +1 @@
+https://drive.google.com/file/d/1FJUNL5h625V9uJc6L4QgH-oLdAOoCixE/view?usp=sharing
```

Comment:
Nice work Johnie. Please move this to an actual Google docs document? That will enable us to comment.